addon/components/time-series-chart.js

```diff
@@ -58,6 +58,12 @@ const TimeSeriesChartComponent = ChartComponent.extend(LegendMixin,
   // Bar left offset, as fraction of width of bar
   barLeftOffset: 0.0,
 
+  // Force X-Axis labels to print vertically
+  xAxisVertLabels: false,
+
+  // Enable smart turning if feasible - TRUMPED by xAxisVertLabels
+  xAxisSmartLabels: false,
+
```

Comment:
I would combine these into a single property, `xAxisVerticalLabels` - which would have string values of `never`, `when-needed`, and `always`, or similar.